packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -89,10 +133,17 @@ class TextEditingController extends ValueNotifier<TextEditingValue> {
   /// that they need to update (it calls [notifyListeners]). For this reason,
   /// this value should only be set between frames, e.g. in response to user
   /// actions, not during the build, layout, or paint phases.
+  ///
+  /// This property can be set from a listener added to this
+  /// [TextEditingController] however one should not also set [selection].
```

Comment:
It feels like there is a full stop or maybe comma missing before "however"?

Reply:
Apparently the pedants (their word, not mine) would have us put a semicolon before the "however" and a comma afterwards: http://www.arrantpedantry.com/2012/02/07/however/.  I will obey; however, I am confident that I will never understand all of the rules for commas.